packages/mobile/src/home/WalletHome.tsx

```diff
@@ -72,28 +78,18 @@ const mapStateToProps = (state: RootState): StateProps => ({
 
 const AnimatedSectionList = Animated.createAnimatedComponent(SectionList)
 
-const HEADER_FADE_HEIGHT = 100
-
-export class WalletHome extends React.Component<Props> {
-  animatedValue: Animated.Value
-  headerOpacity: Animated.AnimatedInterpolation
-  onScroll: () => void
+export class WalletHome extends React.Component<Props, State> {
+  state = {
+    isScrolled: false,
+  }
+  onScroll: (event: NativeSyntheticEvent<NativeScrollEvent>) => void
 
   constructor(props: Props) {
     super(props)
 
-    this.animatedValue = new Animated.Value(0)
-    this.headerOpacity = this.animatedValue.interpolate({
-      inputRange: [0, HEADER_FADE_HEIGHT],
-      outputRange: [0, 1],
-      extrapolate: 'clamp',
-    })
-    this.onScroll = Animated.event(
-      [{ nativeEvent: { contentOffset: { y: this.animatedValue } } }],
-      {
-        useNativeDriver: true,
-      }
-    )
+    this.onScroll = (event) => {
+      this.setState({ isScrolled: event.nativeEvent.contentOffset.y > 0 })
+    }
```

Comment:
Maybe this is fine, but this is gonna call `setState` every 50 ms when you scroll.
It would be more performant to use an `Animated.Value` for this.